cypress/integration/purchase_order_spec.js

```diff
@@ -13,15 +13,12 @@ describe('purchase order Test', function() {
 
             cy.writeIntoStringField('POReference', poReference);
 
-            cy.writeIntoCompositeLookupField('C_BPartner_ID','G0002', 'Test Lieferant')
-            // cy.get('.input-dropdown-container .raw-lookup-wrapper')
-            //     .first()
-            //     .find('input')
-            //     .clear()
-            //     .type('G0002');
-            // these two shall work as of issue https://github.com/metasfresh/metasfresh-webui-frontend/issues/1981
-            //cy.writeIntoCompositeLookupField('C_BPartner_ID', 'Test L', 'Test Lieferant');
-            //cy.writeIntoCompositeLookupField('C_BPartner_Location_ID', 'Testaddresse');
+            cy.writeIntoCompositeLookupField('C_BPartner_ID','G0002', 'Test Lieferant');
+
+            // TODO: I don't see a reason why we would like to fill this field, if it's pre-filled
+            // automatically. But maybe I'm missing something here.  - Kuba
```

Comment:
But in case of this selected BP there's only one address available. And as far as I can tell, there's no way of adding new one from this place of the app.

Reply:
agree, this is not the best test case to test-drive the `writeIntoCompositeLookupField` command...
let me try to extend this spec by first adding another vendor with two contacts...after that I'm going to nag you on how to extract the two into kindof a DSL

Reply:
btw, here https://github.com/metasfresh/metasfresh-webui-frontend/pull/2074/files#diff-50194905bcf3127673a24dcd32cf8fa9R21 I'm selecting BP contact, which is exactly the same as selecting address from this field.

Reply:
Hi, @siemiatj 
i extended purchase_order_spec; can you please take a look at the failing part about creating a C_Location?
note that there is also SQL to be rolled out, in order to select those tabs.
i'm going to roll it out to w101..